crypto/modes/asm/ghash-riscv64.pl

```diff
@@ -1,11 +1,13 @@
 #! /usr/bin/env perl
-# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 2023 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
This should be 2022-2023.  More generally, there is no need to touch this, we've scripts that do it at release time.

Reply:
Will drop that.